hosting/hosting.properties

```diff
@@ -19,3 +19,7 @@ COUCH_DB_PORT=4005
 REDIS_PORT=6379
```

Comment:
probably shouldn't update this file anymore since its deprecated, plan to remove soon!